src/app/item-page/simple/field-components/file-section/file-section.component.html

```diff
@@ -1,4 +1,4 @@
-<ds-metadata-field-wrapper [label]="label | translate">
+<ds-metadata-field-wrapper *ngIf="(files | async).length > 0" [label]="label | translate">
```

Comment:
It's probably safer to add a question mark after the right bracket, to ensure it doesn't break if `(files | async)` returns null or undefined for any reason.
```typescript
(files | async)?.length
```

Reply:
Done!